src/app/home/pages/PlansPage/components/Wizard/NamespacesTable.tsx

```diff
@@ -97,9 +97,24 @@ const NamespacesTable: React.FunctionComponent = () => {
     const editedNamespace = values.editedNamespaces.find(
       (editedNS) => editedNS.id === namespace.id
     );
+    const { currentPlan } = useSelector((state: DefaultRootState) => state.plan);
+    let targetNamespaceFromPlan = '';
+    if (currentPlan) {
+      for (const i of currentPlan.spec.namespaces) {
+        const ns = i.split(':');
+        if (ns.length > 1) {
+          if (ns[0] == namespace.name) {
```

Comment:
These two conditions could be combined into one if statement with &&.

Can you also replace the == with === ? If that's not in our eslint rules I should add it